docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[configure-endpoint-integration-policy]]
+= Configure an integration policy
+
+After the {agent} is installed successfully with {endpoint-sec}, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (a Platinum+ license feature). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, and trusted applications to meet your organization's security needs.
+
+To configure the integration policy:
+
+1. In the {security-app}, go to **Manage** -> **Endpoints** to view the Endpoints list. Remember that you must have admin permissions in {kib} to access this page.
+2. From the **Policy** column, select the integration policy you want to configure. The integration policy configuration page appears.
+3. Review the following settings on the **Policy settings** tab and configure as appropriate:
+* <<malware-protection>>
+* <<ransomware-protection>>
+* <<memory-protection>>
+* <<behavior-protection>>
+* <<event-collection>>
+* <<register-as-antivirus>>
+* <<adv-policy-settings>>
+* <<save-policy>>
+
+4. Click the **Trusted applications** tab to review the list of <<trusted-apps-ov, trusted applications>> assigned to this integration policy. On this tab, you can:
+* Expand and view a trusted application — Click the arrow next to the application's name.
+* View full details for a trusted application — Click the actions button (**...**), then select **View full details**.
+* Unassign a trusted application (Platinum+ subscription) — Click the actions button (**...**), then select **Remove from policy**. This does not delete the trusted application; this just unassigns it from the current policy.
+* Assign an existing trusted application (Platinum+ subscription) — Click **Assign trusted application to policy**, then select an application from the Assign trusted applications flyout. This view lists any existing trusted applications that aren't already assigned to the current policy.
+
+NOTE: To create a new trusted application, go to **Manage** -> **Trusted applications**. You can't create new trusted applications while configuring an integration policy.
```

Comment:
This is trying to preempt readers who might think that, with all the other actions they can perform with TA's here, they can create a new TA. I can see now that it could be a little jarring -- I wonder if swapping the sentence order would help, to put the more direct information first:
```suggestion
NOTE: You can't create new trusted applications while configuring an integration policy. To create a new trusted application, go to **Manage** -> **Trusted applications**.
```